document/accountManager.md

```diff
@@ -82,7 +82,7 @@ client.getHealthFactor(address); //You may check any address
 ### Supply/Withdraw/Borrow/Repay/Liquidate/ClaimRewards
 You may Simply input 'NAVX' as a string or NAVX as a type imported from address.ts. 
 
-Current These Pools are supported: **Sui | NAVX | vSui | USDC | USDT | WETH | CETUS | haSui**
+Current These Pools are supported: **Sui | NAVX | vSui | USDC | wUSDC | USDT | WETH | CETUS | haSui | AUSD | WBTC**
```

Comment:
USDC will be deprecated in subsequent versions, and developers using this interface will need to do a USDC -> wUSDC migration.